src/components/income/F1099Info.tsx

```diff
@@ -309,17 +309,17 @@ export default function F1099Info(): ReactElement {
   const divFields = (
     <Grid container spacing={2}>
       <LabeledInput
-        label="Total Dividends"
+        label={boxLabel('1a', 'Total Dividends')}
         patternConfig={Patterns.currency}
         name="dividends"
       />
       <LabeledInput
-        label="Qualified Dividends"
+        label={boxLabel('1b', 'Qualified Dividends')}
         patternConfig={Patterns.currency}
         name="qualifiedDividends"
       />
       <LabeledInput
-        label="Total capital gains distributions"
+        label={boxLabel('2b', 'Total capital gains distributions')}
```

Comment:
Looks like total capital gains distributions is 2a rather than 2b
![image](https://user-images.githubusercontent.com/8848557/157948599-15c04804-eed5-4e21-ac79-b67bc73b22e4.png)

Reply:
Yikes, good catch.
```suggestion
        label={boxLabel('2a', 'Total capital gains distributions')}
```